nixos/modules/services/networking/unbound.nix

```diff
@@ -266,21 +265,20 @@ in {
         ProtectControlGroups = true;
         ProtectKernelModules = true;
         ProtectSystem = "strict";
+        ProtectClock = true;
+        ProtectHostname = true;
+        ProtectProc = "invisible";
+        ProcSubset = "pid";
+        ProtectKernelLogs = true;
+        ProtectKernelTunables = true;
         RuntimeDirectory = "unbound";
         ConfigurationDirectory = "unbound";
         StateDirectory = "unbound";
         RestrictAddressFamilies = [ "AF_INET" "AF_INET6" "AF_NETLINK" "AF_UNIX" ];
         RestrictRealtime = true;
         SystemCallArchitectures = "native";
         SystemCallFilter = [
-          "~@clock"
-          "@cpu-emulation"
-          "@debug"
-          "@keyring"
-          "@module"
-          "mount"
-          "@obsolete"
-          "@resources"
+          "~@clock @cpu-emulation @debug @keyring @module @mount @obsolete @resources @privileged"
```

Comment:
What about starting with an allow-list `@system-service` which is recommended by systemd?

Reply:
Good point. I have tested that just `@system-service` is sufficient (usage of two privileges doesn't involve special syscall)